extensions/amp-ad-network-cloudflare-impl/0.1/cloudflare-a4a-config.js

```diff
@@ -16,11 +16,13 @@
 
 /**
  * Determines which tags desire A4A handling
- *
+ * @param {!Window} win
```

Comment:
The unused variables might need to be prefaced with `unused`, or type checker might complain.

Reply:
no complaints from the type checker